src/utils.js

```diff
@@ -669,6 +670,60 @@ export const parseContentTypeFromResponse = (response) => {
     };
 };
 
+/**
+ *
+ * @param response
+ * @return {undefined|Array}
+ */
+export const getCookiesFromResponse = (response) => {
+    const { headers } = response;
+    let cookies;
+
+    if (Array.isArray(headers['set-cookie'])) {
+        cookies = headers['set-cookie'].map(Cookie.parse);
+    } else {
+        cookies = [Cookie.parse(headers['set-cookie'])];
+    }
+
+    return cookies;
+};
+
+/**
+ * Gets cookie string for the "Cookie" header
+ * @param cookies {Array}
+ * @return {String|undefined}
+ */
+export const getCookieHeader = (cookies) => {
```

Comment:
It expects cookies of type `Cookie` so it's not a general purpose utility function, it can be part of `Session` or `SessionPool` internally.